.pylintrc

```diff
@@ -0,0 +1,11 @@
+[MASTER]
+# Go as fast as you can
+jobs = 0
+
+[MESSAGES CONTROL]
+disable = all
```

Comment:
Why disable all?

Reply:
we don't enforce pylint rules in this repo (yet)